### PR TITLE
clean up and reorganize folders and files

### PR DESCRIPTION
**Pull request by ycao56**
1. moved all server side code to `server` folder
2. moved all client side code to `src-web` folder
3. fixed alert styling issue
image

image

### CODE REVIEW

**Review by ChunxiAlexLuo — APPROVED** [review]
lgtm

File: src-web/components/modals/RemoveResourceModal.js [hunk 1]
```diff
@@ -149,8 +149,9 @@ class RemoveResourceModal extends React.Component {
             {reqStatus === REQUEST_STATUS.ERROR &&
               <AcmAlert
                 variant='danger'
-                title=''
-                subtitle={reqErrorMsg || msgs.get('error.default.description', locale)} />}
+                isInline={true}
+                noClose={true}
```
Comment (dhaiducek) [inline src-web/components/modals/RemoveResourceModal.js:153]: I tried fixing this, too. I had to add both:
```
noClose={true}
canClose={false}
```

Comment (ycao56) [reply src-web/components/modals/RemoveResourceModal.js:153]: doesn't look like AcmAlert have props `canClose` https://github.com/open-cluster-management/ui-components/blob/main/src/AcmAlert/AcmAlert.tsx#L102-L110

Comment (dhaiducek) [reply src-web/components/modals/RemoveResourceModal.js:153]: Yup--nevermind. I must have been confusing them--it works.

**Review by dhaiducek — APPROVED** [review]

**Pull request merged**